iroha-cpp/main.cpp

```diff
@@ -112,9 +112,22 @@ iroha::protocol::Transaction generateTransaction(const std::string& account_name
 }
 
 
-iroha::protocol::Transaction sendSampleTransaction(const std::string& account_name, const std::string& key_path, const std::string& peer_ip, int torii_port, uint32_t quorum, const std::string& domain_id, const std::string& user_default_role, const std::string& asset_name)
+iroha::protocol::Transaction sendSampleTransaction(const std::string& account_name,
+                                                   const std::string& key_path,
+                                                   const std::string& peer_ip,
+                                                   int torii_port, uint32_t quorum,
```

Comment:
Port numbers are limited ([according to wikipedia](https://en.wikipedia.org/wiki/Port_(computer_networking)#Port_number)) to: "A port number is a 16-bit unsigned integer, thus ranging from 0 to 65535", so I suggest:
`int` -> `uint16_t`

Reply:
Yes. This is the port range.